src/domain/ledger/index.types.d.ts

```diff
@@ -44,7 +44,6 @@ type LedgerTransaction = {
   readonly feeUsd: number
 
   // for IntraLedger
-  readonly walletPublicId?: WalletPublicId
```

Comment:
This was originally meant to be the recipient's `walletPublicId`. Should we instead rename this to something like `recipientWalletId` instead of removing?

Reply:
does this resolves your concern: https://github.com/GaloyMoney/galoy/pull/792/commits/82d3cc1ecb878e975215fd3ea6326a6d566da18b ?